worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -561,6 +561,17 @@ private[celeborn] class Worker(
     sb.toString()
   }
 
+  override def decommission: String = {
+    exitKind = CelebornExitKind.WORKER_DECOMMISSION
+    new Thread() {
+      override def run(): Unit = {
+        Thread.sleep(30000)
```

Comment:
Emm.. why should defer to trigger the exit?

Reply:
Hmm for http response.. so add this delay, we can set it to other value, any idea?

Reply:
30s is too long, Maybe 10s

Reply:
Done

Reply:
@waitinfuture see here